src/components/domain/survey/answerType/TextAnswer.tsx

```diff
@@ -0,0 +1,48 @@
+import { useFormContext } from 'react-hook-form';
+import { TextAnswerProps } from '@/models/survey/surveyModels';
+import SurveyLayout from '../SurveyLayout';
+import TextRow from '../answerRow/TextRow';
+
+interface InstructionProps {
+  type: '강점' | '보완점';
+}
+
+const Instruction: React.FC<InstructionProps> = ({ type }) => (
```

Comment:
```suggestion
const Instruction = ({ type }:InstructionProps) => (
```

React.FC로 타입 지정해도 되지만, 그동안 저희 컨벤션과는 조금 상이한 것 같아서 언급드립니다

Reply:
✅ [a239fe4](https://github.com/PRism-Works/PRism-FE/pull/92/commits/a239fe4c9a467a2dab0b499a5303905fa9b5fb68)